### PR TITLE
added json for profile at LinkFree

### PR DESCRIPTION
**Pull request by diva-21**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2348"></a>

### CODE REVIEW

**Comment by diva-21** [comment]
Can i know whats the error in the pull request? @EddieHubCommunity/ambassadors

File: data/diva-21.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "url": "https://hashnode.com/@diva1221",
+      "icon": "hashnode"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/l-divakar-102205215/",
+      "icon": "linkedin"
+    }
+  ]
+}
```
Comment (Pradumnasaraf) [inline data/diva-21.json:28]:
```suggestion
}

```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, just an ending line is required

**Comment by diva-21** [comment]
> Hey, just an ending line is required

thank you, @Pradumnasaraf, I didn't check the minute part

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good to me! 👍

**Comment by diva-21** [comment]
are these changes enough ?

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by diva-21** [comment]
> Looks good to merge. Thank you for your contribution! 🎉

Thank you, @Pradumnasaraf

**Pull request merged**